lib/src/virtual_host.dart

```diff
@@ -73,14 +73,14 @@ class _VirtualHost implements VirtualHost {
         return;
       }
       var domains = host.split('.');
-      var current = _topDomain;
-      StreamController any;
+      _VirtualHostDomain? current = _topDomain;
```

Comment:
Make this non-nullable and then do the assertion at the assignment below:

```dart
current = current.subdomains[domains[i]]!;
```

Reply:
Done.